### ISSUE TITLE
Error on build (gongular changed functions declaration)

### ISSUE THREAD

**Issue by ghost** [smallnest/go-web-framework-benchmark#40]
The command `go build -o  gowebbenchmark server.go` could not be executed because of gongular framework changed some functions declarations.

**Comment by razonyang** [comment]
The following command works for me.

```
cd $GOPATH/src/github.com/mustafaakin/gongular
git checkout v1.0
```

**Comment by ghost** [comment]
Thanks @razonyang! But I've tried to update the code and I've published a pull request (see #41).

**Comment by smallnest** [comment]
fixed and added.
